packages/react-button/src/components/Button/Button.types.ts

```diff
@@ -87,61 +89,59 @@ export type ButtonProps = ComponentProps &
     /** A button can be formatted to show only text in order to indicate a less-pronounced action. */
     // text?: boolean;
 
-    tokens?: RecursivePartial<ButtonTokenSet>;
+    tokens?: RecursivePartial<ButtonTokens>;
   };
 
 export interface ButtonState extends ButtonProps {
   buttonRef?: React.RefObject<HTMLButtonElement>;
 }
 
-export interface ButtonSlots extends BaseSlots {
-  icon: React.ElementType;
-  loader: React.ElementType;
-}
-
-export type ButtonSlotProps = SlotProps<ButtonSlots, ButtonProps, React.ButtonHTMLAttributes<HTMLButtonElement>>;
-
-export type ButtonTokenSet = ColorTokenSet & {
-  /* sizing */
-  paddingLeft: string;
-  paddingRight: string;
-  paddingTop: string;
-  paddingBottom: string;
-  margin: string;
-  height: string;
-  minWidth: string;
-  maxWidth: string;
-  minHeight: string;
-  contentGap: string;
-  iconSize: string;
-  borderRadius: string;
-  borderTopLeftRadius: string;
-  borderTopRightRadius: string;
-  borderBottomLeftRadius: string;
-  borderBottomRightRadius: string;
-  borderWidth: string;
-  boxShadow: string;
-  width: string;
-
-  size: {
-    smallest: string;
-    smaller: string;
-    small: string;
-    regular: string;
-    large: string;
-    larger: string;
-    largest: string;
+export type ButtonTokens = ColorTokens &
+  FontTokens & {
+    /* sizing */
+    paddingLeft?: string;
+    paddingRight?: string;
+    paddingTop?: string;
+    paddingBottom?: string;
+    margin?: string;
+    height?: string;
+    minWidth?: string;
+    maxWidth?: string;
+    minHeight?: string;
+    contentGap?: string;
+    iconSize?: string;
+    borderRadius?: string;
+    borderTopLeftRadius?: string;
+    borderTopRightRadius?: string;
+    borderBottomLeftRadius?: string;
+    borderBottomRightRadius?: string;
+    borderWidth?: string;
+    boxShadow?: string;
+    width?: string;
+
+    transform?: string;
+    transition?: string;
+
+    size?: {
+      smallest?: string;
+      smaller?: string;
+      small?: string;
+      regular?: string;
+      large?: string;
+      larger?: string;
+      largest?: string;
+    };
+
+    pressed?: {
+      transform?: string;
+      transition?: string;
```

Comment:
these fields look outdated

Reply:
hmmm. look right to me?

Reply:
if you compare this with useButtonClasses, tokens such as `whitespace` are missing